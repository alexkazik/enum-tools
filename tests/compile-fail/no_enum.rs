#![no_implicit_prelude]

use ::enum_tools::EnumTools;

#[derive(EnumTools, Debug, Clone, Copy, Eq, PartialEq)]
#[repr(i8)]
pub(crate) struct EG();

fn main() {}
