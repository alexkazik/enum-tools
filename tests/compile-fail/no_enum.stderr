error: only enums are supported
 --> tests/compile-fail/no_enum.rs:6:1
  |
6 | #[repr(i8)]
  | ^

error[E0517]: attribute should be applied to an enum
 --> tests/compile-fail/no_enum.rs:6:8
  |
6 | #[repr(i8)]
  |        ^^
7 | pub(crate) struct EG();
  | ----------------------- not an enum
